pkg/gadgets/trace/capabilities/tracer/tracer_test.go

```diff
@@ -218,6 +218,52 @@ func TestCapabilitiesTracer(t *testing.T) {
 				}
 			},
 		},
+		"unique_false": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
+					Unique:     false,
+				}
+			},
+			generateEvent: repeatChown,
+			validateEvent: func(t *testing.T, info *utilstest.RunnerInfo, _ interface{}, events []types.Event) {
+				nfound := 0
+				for _, event := range events {
+					if event.CapName == "CHOWN" {
+						nfound++
+					}
+				}
+
+				if nfound <= 1 {
+					t.Fatalf("Capability not found")
+				}
+			},
+		},
+		"unique_true": {
+			getTracerConfig: func(info *utilstest.RunnerInfo) *tracer.Config {
+				return &tracer.Config{
+					MountnsMap: utilstest.CreateMntNsFilterMap(t, info.MountNsID),
```

Comment:
Maybe too complicated, but it would be useful also to implement a test where we generate the same capability event from two differente mount namespaces (both included in the `MountnsMap`) and verify that one event is generated per mout namespace.